app/models/monstercard.rb

```diff
@@ -98,4 +98,145 @@ def self.equip_monster(params, player)
     # type = "equip"
     { type: type, message: message }
   end
+
+  def self.bad_things(ingamedeck, gameboard)
+    player = Player.find_by('id = ?', gameboard.current_player)
+
+    case ingamedeck.card.action # get the action from card
+    when 'lose_item_hand'
+      monster_arr = []
+      monster_slot = []
+      if player.monsterone.cards.where(item_category: 'hand').any?
```

Comment:
bietet sie an in a eigene funktion ausgelagert zu werden weils so lang is & so oft verwendet wird